### DISCUSSION TITLE
Stuck trying to access a folder on synology DS version 7.1.1

### DISCUSSION THREAD

**Discussion by Xilr8** [IceWhaleTech/CasaOS#1434] [Q&A]
Hello from a Noob stuck trying to share a folder.

I'm hoping to find some guidance, and apologize if this has been answered ( I did look but couldn't find a solution). I've been using a D214+ hapily for many years. I was recently gifted a mini server to run CasaOS which I plan to use for my plex server as well as hosting images. (I realize my synology can do both of these things very well). What I would like to do is use the synology for storage but CasaOS to run some services to share the burden. When I mount the shared folder in CasaOS using it's smb address, I'm able to see the folders I'm looking for but not the contents, and I'm not able to create or modify anything in those folders, they look blank.

I remember having a similar issue setting up tranmission on an old WD nas as well, but don't remember how I got it working.

Windows file explorer works fine I'm able to see, move, add or delete as needed.

I believe it's a permissions issue but I can't for the life of me figure out what the problem is. On the DS214+ I've given permission to read and write under "edit shared folder" I've tried it with and without the advance share permission enabled.

Any help would be appreciated.

util@util-server-01:~$ sudo mkdir /mnt/video
[sudo] password for util: 
mkdir: cannot create directory ‘/mnt/video’: File exists
util@util-server-01:~$ sudo mkdir /mnt/volume1/video
mkdir: cannot create directory ‘/mnt/volume1/video’: No such file or directory

![CasaOS showing plex folders](https://github.com/IceWhaleTech/CasaOS/assets/146789765/3cc112e3-4122-4c40-bd67-54f61244e9e0)
![unable to see contents](https://github.com/IceWhaleTech/CasaOS/assets/146789765/b147c5d3-7f76-4063-8cc1-cf14401bcadc)

**Comment by torreluca** [comment]
Hi, I have the same problem but I see that the community is not helpful at all. Were you able to solve it?

**Reply by MichalCRP** [reply]
Same problem and no solution found anywhere

**Comment by misaelr78** [comment]
Im having exactly the same issue. No content inside any folder.

**Comment by rttm70** [comment]
same same - ![image](https://github.com/IceWhaleTech/CasaOS/assets/76599588/7536e454-97ba-4e94-9aa7-d0fe5f2d1991) 
no content in the folder visible ...

**Comment by Agent772** [comment]
Same problem, any solutions on this?
otherwise ill have to uninstall casaOS
